mri_segment_hypothalamic_subunits/mri_segment_hypothalamic_subunits

```diff
@@ -1234,14 +1231,15 @@ def list_subfolders(path_dir, whole_path=True, expr=None, cond_type='or'):
 
 def mkdir(path_dir):
```

Comment:
A more pythonic way of making recursive directories would be to use:

def mkdir(path_dir):
    os.makedirs(path_dir, exist_ok=True)

Reply:
yes I agree, but this function is from a long time ago, and using os.makedirs consistently across all the repos would take me quite some time.
The main thing for this PR is really to fix a problem with the cortical parcellation volumes, so it's quite urgent. The sooner the better :)
Thanks !
Benjamin